book/src/appendices/a_config_files/ball_config.md

```diff
@@ -0,0 +1,127 @@
+# Adding a Ball Config
+For simplicity, we will wrap all of our Config objects into a single `PongConfig` object backed by a single 
+`config.ron` file, but know that you can just as easily keep them in separate files and read from each file 
+separately.
+
+The `BallConfig` will replace the `BALL_VELOCITY_X`, `BALL_VELOCITY_Y`, `BALL_RADIUS`, and `BALL_COLOR` 
+variables. However, the `BALL_COLOR` was originally an array, but Ron does not currently support those, so it
```

Comment:
Might be more accurate to say "Serde does not currently support those."

Ron supports the full serde data model, fixed length arrays aren't included in that unfortunately.

Reply:
Serde and RON both do, they're handled just like tuples.